core/chains/evm/config/toml/defaults/Blast_Mainnet.toml

```diff
@@ -0,0 +1,20 @@
+ChainID = '81457'
+FinalityDepth = 200
+FinalityTagEnabled = true
+ChainType = 'optimismBedrock'
+
+[GasEstimator]
+EIP1559DynamicFees = true
+BumpThreshold = 60
+BumpPercent = 20
+BumpMin = '100 wei'
+PriceMax = '120 gwei'
+LimitDefault = 8000000
+FeeCapDefault = '120 gwei'
+
+[HeadTracker]
+HistoryDepth = 300
```

Comment:
HistoryDepth is no longer needed to be larger than `FinalityDepth` https://github.com/smartcontractkit/chainlink/pull/13494 . Perhaps we want to set this to something smaller?